### ISSUE TITLE
Images with invalid src on BookTile are not pulling custom missing img

### ISSUE THREAD

**Issue by Qtipshooter** [Qtipshooter/Bookstore#2]
Images with invalid src on BookTile are not pulling custom missing img, they are just putting a blank no image and showing alt text
